pandas/io/tests/parser/test_unsupported.py

```diff
@@ -20,6 +20,16 @@
 
 
 class TestUnsupportedFeatures(tm.TestCase):
+    def test_mangle_dupe_cols_false(self):
+        # see gh-12935
+        data = 'a b c\n1 2 3'
+        msg = 'is not supported'
+
+        for engine in ('c', 'python'):
+            with tm.assertRaisesRegexp(ValueError, msg):
```

Comment:
oh its here. But does this depend on whether ``names`` is passed and/or are dupes?